content/applications/inventory_and_mrp/manufacturing/quality_control/instructions_check.rst

```diff
@@ -0,0 +1,85 @@
+==========================
+Instructions quality check
+==========================
+
+.. |QCP| replace:: :abbr:`QCP (Quality Control Point)`
+
+In Odoo *Quality*, an Instructions check is one of the quality check types that can be selected when
+creating a new quality check or Quality Control Point (QCP). Instructions checks consist of a text
+entry field that allows the creator to provide instructions for how to complete the check.
+
+There are two distinct ways that quality checks can be created. A single check can be manually
+created. Alternatively, a |QCP| can be configured that will automatically create checks at a
+predetermined interval.
+
+This documentation will only detail the configuration options and processing steps that are unique
+to Instructions quality checks. For a full overview of all the configuration options available when
+creating a single check or a |QCP|, see the documentation on :ref:`Quality checks
+<manufacturing/quality_control/quality-checks>` and :ref:`Quality Control Points
+<manufacturing/quality_control/quality-control-points>`.
+
+Create an Instructions quality check
+====================================
+
+Instructions quality checks can be created by navigating to :menuselection:`Quality --> Quality
+Control --> Quality Checks`, and clicking :guilabel:`New`. It is also possible to create a |QCP|
+that is configured to automatically create an Instructions quality check when
+certain criteria is met. To do so, navigate to :menuselection:`Quality --> Quality Control -->
+Control Points`, and click :guilabel:`New`.
+
+On the quality check or |QCP| form, select :guilabel:`Instructions` from the :guilabel:`Type`
+drop-down field. If a single quality check is being created, enter the instructions for how to
+complete the check in the :guilabel:`Instructions` text entry field of the :guilabel:`Notes` tab at
+the bottom of the form. If a |QCP| is being configured, enter the instructions for how to complete
+the checks it creates in the text-entry field of the :guilabel:`Instructions` tab at the bottom of
+the form.
+
+Fill out the rest of the required fields for the quality check or |QCP|, as detailed in the
+documentation for :ref:`Quality checks <manufacturing/quality_control/quality-checks>` and
+:ref:`Quality Control Points <manufacturing/quality_control/quality-control-points>`. Finally, click
+:guilabel:`Save` to create the quality check or |QCP|.
+
+Process an Instructions quality check
+=====================================
+
+To process an Instructions quality check, select a manufacturing order or an inventory order
```

Comment:
What you had previously is totally fine, I've just never seen this term used in the inventory docs before. I've more commonly seen: 
```suggestion
To process an Instructions quality check, select a manufacturing order or an warehouse transfer
```
Just wondering if we should follow what others have been doing (but we're writing for inventory, maybe *we're* the ones coming up with terminology!! :brain: